editoast/schemas/src/rolling_stock.rs

```diff
@@ -156,16 +156,25 @@ impl<'de> Deserialize<'de> for RollingStock {
             .supported_signaling_systems
             .0
             .contains(&"ETCS_LEVEL2".to_string())
-            && rolling_stock.etcs_brake_params.is_none()
         {
             return Err(serde::de::Error::custom(
-                "invalid rolling-stock: supporting ETCS_LEVEL2 signaling system requires providing ETCS brake parameters.",
+                "invalid rolling-stock: ETCS_LEVEL2 requires 'etcs_brake_params' field instead of 'ETCS_LEVEL2'.",
```

Comment:
I'd go for longer but more explicit:
```suggestion
                "invalid rolling-stock: 'ETCS_LEVEL2' can't be listed in 'supported_signaling_systems', providing 'etcs_brake_params' field is the (only) way to trigger ETCS_LEVEL2 support.",
```